mavo-sort.js

```diff
@@ -0,0 +1,73 @@
+(function() {
+
+Mavo.Plugins.register("sort", {});
+
+Mavo.Functions.sort = function(array, ...properties) {
+	var arrayCopy = array.slice();
+	var splitChar = ".";
```

Comment:
Hmm I guess it was mostly because I wasn't sure if the dot notation was the actual way that nested properties were resolved, though, in retrospect this only makes my job easier as the programmer to change it if I happened to be wrong, plus I have to change it to the correct way in the end anyway, so will do